price_parser/parser.py

```diff
@@ -262,13 +267,20 @@ def parse_number(num: str) -> Optional[Decimal]:
     Decimal('1235.99')
     >>> parse_number("1.235€99")
     Decimal('1235.99')
+    >>> parse_number("140.000", decimal_separator_hint=",")
+    Decimal('140000')
+    >>> parse_number("140.000", decimal_separator_hint=".")
+    Decimal('140.000')
     >>> parse_number("")
     >>> parse_number("foo")
     """
     if not num:
         return None
     num = num.strip().replace(' ', '')
-    decimal_separator = get_decimal_separator(num)
+    decimal_separator = (
```

Comment:
decimal_separator = decimal_separator or get_decimal_separator(num)

Reply:
I made some changes of [regexp](https://github.com/scrapinghub/price-parser/pull/18/files#diff-1bcf8c415357b577173d2e003a172525R191)  in `extract_price_text` and added few [tests](https://github.com/scrapinghub/price-parser/pull/18/files#diff-091825c38112ff1dafc081caf2916d27R1965) which cover this. It should fix 1 case with euro.